api/managementpb/alerting/alerting.proto

```diff
@@ -1,15 +1,15 @@
 syntax = "proto3";
 
-package ia.v1beta1;
+package alerting.v1beta1;
```

Comment:
btw we'll need to drop the `v1beta1` package post GA?

Reply:
Yep, removed this `beta`.